tools/buildtools/ambertools/tleap.xml

```diff
@@ -0,0 +1,2043 @@
+<tool id="tleap" name="Build tLEaP" version="0.0.1" >
+    <description>interactively build and run tLEaP files to set up systems with AmberTools</description>
+     <macros>
+        <import>macros.xml</import>
+        <token name="@GALAXY_VERSION@">0</token>
+     </macros>
+     <expand macro="requirements" />
+     <stdio>
```

Comment:
remove this stdio and replace the functionality with detect_errors https://docs.galaxyproject.org/en/latest/dev/schema.html#id24